src/schedule/components/ScheduleTab.tsx

```diff
@@ -0,0 +1,41 @@
+// 일정 탭 컴포넌트
+import React from "react";
+
+interface ScheduleTab {
+  tabTitle: string;
+  tabItems: Record<"title", string>[];
+  currentTab: string;
+  onClickTab: (tab: string) => void;
+}
+
+const ScheduleTab = ({
+  tabTitle,
+  tabItems,
+  currentTab,
+  onClickTab,
+}: ScheduleTab) => {
+  return (
+    <>
+      <div className="text-zinc-800 text-[32px] font-bold capitalize">
+        {tabTitle}
+      </div>
+      <div className="flex mt-9 gap-x-10">
+        {tabItems.map((tab, i) => (
```

Comment:
```suggestion
        {tabItems.map(({title}, i) => (
```

가독성을 위한 취향차이긴하지만 map 매개변수를 구조분해로 받는다면 아래 코드들에서 `tab.title`을 title로만 사용할수 있습니다!